content/source/assets/stylesheets/_docs.scss

```diff
@@ -1,26 +1,45 @@
 #docs-sidebar {
   margin-bottom: 30px;
-  margin-top: 20px;
   overflow: hidden;
 
-  h1,
-  h2,
+  // Re-using the .nav class name on lists means we inherit some
+  // unfortunate behavior from Bootstrap, so reset all that.
+  .nav {
+    ul, li, a {
+      float: none;
+      position: static;
+      padding: 0;
+    }
+    ul { display: block; }
+    li { display: list-item; }
+    a { display: inline; }
+  }
+
+  // hide an outdated implementation of expand/collapse link
+  a.subnav-toggle {
+    display: none;
+  }
+
+  a#mobile-skip {
+    color: $body-link-color;
+  }
+
   h3,
-  h4,
-  h5,
-  h6 {
-    margin-top: 10px;
+  h4 {
+    min-height: 35px;
+    margin-top: 30px;
+    display: flex;
+    align-items: center;
```

Comment:
I can't see why we need flex here. Is there some page with multiple children in the `h4`?

Reply:
Nope — this is a workaround to get the H1 in the main area and the H4 in the sidebar aligned Just So. I'm open to doing something else, but if I'm remembering the eighty things I tried on Friday correctly, it goes something like:

- I need to line up those two headers _at their text baselines,_ and I also need to line up the controls beneath them. I can't use grid or anything, so I have to do px math and make sure it adds up to the same things.
- Lining up the content boxes of the controls works fine because their texts use the same font metrics... but the headers look way off if I do that.
- CSS doesn't give me any way to specify where the text baseline should be for a line box inside a block box... I think. 
- Aligning the header boxes slightly askew ends up making it harder to align the controls beneath. 
- Well, flexbox actually lets me center things vertically, or say "text starts at the bottom of the box, not the top". And if I put the boxes _like so,_ the baselines look like they line up if I center the text in the H4 box and flex-end the text in the H1 box. 

I recognize that's kind of a crime, but I was starting to run out of ways to align things. idk, any ideas?

Reply:
Oh, and a related question — I just remembered that we have some rare H1 headers that go to 2 lines. Should I leave enough room in the layout for those? That ends up looking like this: 

![Screen Shot 2019-06-10 at 2 34 40 PM](https://user-images.githubusercontent.com/484309/59228571-fbeff480-8b8c-11e9-87df-ab303e79230c.png)

![Screen Shot 2019-06-10 at 2 35 15 PM](https://user-images.githubusercontent.com/484309/59228581-027e6c00-8b8d-11e9-9999-544e8402454d.png)